common/artifacts/000_wc_artifacts.txt

```diff
@@ -562,12 +668,195 @@ runeblade_unholy_1 = {
 	}
 
 	martial = 1
-	combat_rating = 5
+	combat_rating = 6
+	
+	monthly_character_prestige = 0.15
+	monthly_character_piety = 0.05
+
+	command_modifier = {
+		morale_offence = 0.05
+	}
 
 	picture = "GFX_runeblade_unholy"
 	slot = weapon
 }
 
+### One-handed weapon ###
+
+# Legendary
+felomelorn = {
+	quality = 5
+	flags = { combat_weapon sword }
+
+	active = {
+		is_adult = yes
+		is_physical_class_trigger = yes
+	}
+	allowed_gift = {
+		always = no
+	}
+
+	stacking = no
+	indestructible = yes
+
+	diplomacy = 1
+	learning = 2
+	combat_rating = 10
+
+	monthly_character_prestige = 0.8
+	monthly_character_piety = 0.35
+	
+	general_opinion	= 5
+	cult_of_sunwell_opinion = 5
+
+	picture = "GFX_felomelorn"
+	slot = weapon
+}
+serathil = {
+	quality = 5
+	flags = { combat_weapon axe }
+
+	active = {
+		is_adult = yes
+		is_physical_class_trigger = yes
+	}
+	allowed_gift = {
+		always = no
+	}
+
+	stacking = no
+	indestructible = yes
+
+	martial = 2
+	combat_rating = 25
+
+	monthly_character_prestige = 1.5
+	
+	orcish_shamanism_opinion = 10
+
+	picture = "GFX_serathil"
+	slot = weapon
+}
+sulfuras = {
+	quality = 5
+	flags = { combat_weapon axe }
+
+	active = {
+		is_adult = yes
+		is_physical_class_trigger = yes
+		OR = {
+			trait = giant
+			trait = strong
+			trait = robust
+			is_large_humanoid_trigger = yes
+		}
+		combat_rating = 50
+	}
+	allowed_gift = {
+		always = no
+	}
+
+	stacking = no
+	indestructible = yes
+
+	martial = 2
+	intrigue = 1
+	learning = 1
+
+	health = 0.5
+	combat_rating = 25
+
+	monthly_character_prestige = 0.7
+	monthly_character_piety = 0.4
+
+	picture = "GFX_sulfuras"
+	slot = weapon
+}
+hammer_of_the_twilight = {
+	quality = 5
+	flags = { combat_weapon hammer }
+
+	active = {
+		is_adult = yes
+		OR = {
+				is_magic_class_trigger = yes
+				is_physical_class_trigger = yes
+		}
+		religion = old_gods_worship
+	}
+	allowed_gift = {
+		always = no
+	}
+
+	stacking = no
+	indestructible = yes
+
+	martial = 1
+	intrigue = 1
+	learning = 2
+	combat_rating = 20
+
+	monthly_character_prestige = 0.7
+	monthly_character_piety = 0.4
+	void_group_opinion = 10
+
+	picture = "GFX_hammer_of_the_twilight"
+	slot = weapon
+}
+thunderfury = {
+	quality = 5
+	flags = { combat_weapon sword void_artifact}
+
+	active = {
+		is_adult = yes
+		is_physical_class_trigger = yes
+		}
+
+	allowed_gift = {
+		always = no
+	}
+
+	stacking = no
+	indestructible = yes
+
+	martial = 2
+	health = 1
+	combat_rating = 20
+
+	monthly_character_prestige = 1.5
+
+	picture = "GFX_thunderfury"
+	slot = weapon
+}
+
+# Epic
+ironfoe = {
+	quality = 4
+	flags = { combat_weapon hammer }
+	
+	active = {
+		is_adult = yes
+		is_physical_class_trigger = yes
+	}
+
+	stacking = no
+	diplomacy = 1
+	martial = 1
+	combat_rating = 15
+
+	monthly_character_prestige = 1
+
+	light_group_opinion = 6
+	mystery_of_the_makers_opinion = 6
```

Comment:
Hmm? Ironfoe isn't religious artifact... at all. Moreover, it's created by a dark iron dwarf...
https://wow.gamepedia.com/Franclorn_Forgewright
And the Dark Iron dwarves are basically either Light worshippers of Fire fanatics. Not Titanic.

Reply:
in the timeline, wielded by reginald windsor. " the legendary orc-slaying hammer" <-- that should make light_group like the wielder, right?
regarding the titan worshippers: true, i didn't check different religions of ironforge and shadowforge

Reply:
mystery_of_the_makers changed to fire_cult now.